.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+**What this PR does / why we need it**:
+
+**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
+Fixes #
+
+**Special notes for your reviewer**:
+
+1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.
```

Comment:
I don't understand "changes any image versions".  Can you rephrase?

Reply:
Sorry, I got this template from https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/master/.github/PULL_REQUEST_TEMPLATE.md , but seems this sentence needs to be removed.